.github/workflows/reusable-crn-analytics-algolia-sync.yml

```diff
@@ -118,3 +118,11 @@ jobs:
           ALGOLIA_APP_ID: ${{ steps.algolia-keys.outputs.algolia-app-id }}
           ALGOLIA_INDEX: ${{ steps.setup.outputs.crn-analytics-algolia-index }}
           ALGOLIA_INDEX_TEMP: '${{ steps.setup.outputs.crn-analytics-algolia-index }}_${{ github.run_id }}_temp'
+      - name: Process Productivity Performance
+        if: ${{ inputs.metric == 'all' || inputs.metric == 'team-productivity' || inputs.metric == 'user-productivity' }}
```

Comment:
When running only for team-leadership https://github.com/yldio/asap-hub/actions/runs/9224439267/job/25379705095#step:13:11

![Screenshot 2024-05-24 at 10 12 03](https://github.com/yldio/asap-hub/assets/16595804/eb232520-841e-4cfb-be3d-1aee73edf4df)

When running only for team-productivity https://github.com/yldio/asap-hub/actions/runs/9224927942/job/25381269805#step:15:8

![Screenshot 2024-05-24 at 10 55 26](https://github.com/yldio/asap-hub/assets/16595804/47747028-1540-47ab-a6fa-730050e2494a)